{
    "checksum": "7c1922a92cfb0b33c9f733a45126f28ae27c606f6799e01be3b8ab364cc5e568",
    "file": "InvalidpathLenConstraintTest6EE.p12",
    "mime": {
        "encoding": "binary",
        "type": "application/octet-stream"
    },
    "number_of_duplicates": 6,
    "package": "python-cryptography-vectors",
    "path": "python-cryptography-vectors/3.4.8-1/cryptography_vectors/x509/PKITS_data/pkcs12/InvalidpathLenConstraintTest6EE.p12",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/python-cryptography-vectors/3.4.8-1/",
        "metric": {
            "size": 78852
        },
        "pts_link": "https://tracker.debian.org/pkg/python-cryptography-vectors",
        "sloc": [
            [
                "python",
                61
            ],
            [
                "sh",
                49
            ],
            [
                "makefile",
                3
            ]
        ],
        "suites": [
            "bookworm",
            "sid"
        ],
        "vcs_browser": "https://salsa.debian.org/python-team/packages/python-cryptography-vectors",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/p/python-cryptography-vectors/3.4.8-1/cryptography_vectors/x509/PKITS_data/pkcs12/InvalidpathLenConstraintTest6EE.p12",
    "stat": {
        "perms": "rw-r--r--",
        "size": 2722,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": false,
    "type": "file",
    "version": "3.4.8-1"
}